### DISCUSSION TITLE
nicknames for numbered entities in the recording system

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1236] [users: ideas for new features]
Moved here from "ideas and drafts for the recording system" in the wiki:

At both the unit level and the collected material level people assign something like a nickname, or an alternate way of referring to things, that seems to be so instinctual that fighting it is counter-productive. So we have begun to discuss how to make it regular, so that nicknames are not repeated, for instance. Examples were Leslie's use of Sesame Street names for her units, and Miriam's use of MM numbers for her samples (which she abused - MM4 was used for two URAP numbers, which is precisely why this conversation got started). Our initial thinking is that we should not use a shopping basket method for insuring lack of doubled use, but rather that it is pretty unlikely something would be double assigned and we should ask synchronization to catch it if that happens instead.

**Comment by lbestock** [comment]
And I still agree with that initial thinking. One assigns nicknames because one thinks (sometimes erroneously) that they'll be more intuitively unrepeated than normal numbering systems. A question for this is not just where they are created but also where they are used and displayed. I can imagine a configuration setting that allows for choosing whether or not units show up from the start page as numbers or as nicknames, for instance. Gallery searches. But what about locus lists? You may call the unit Oscar the Grouch, but do you call the locus Oscar the Grouch-003?

**Reply by urapadmin** [reply]
no. The nickname would be like an alternative number. 

"instinctual" does not strike me a valid English word. 

nicknames get bashed here no matter where and when they pop up. Do other archaeologists think of them the same way?

**Reply by urapadmin** [reply]
I mean, we are not trying to be opinionated. We are trying to offer users a system that supports their methods. The question is not: Do we want to use this on Uronarti! Just as a reminder.

**Reply by urapadmin** [reply]
I wish we had more voices (multivocality- who believes in that?) at least in the discussion page 😞

**Reply by lbestock** [reply]
Univocality strikes me as one of the defining features of our dysfunctional era. 

Instinctive? 

And I did not think I was bashing nicknames. Oscar the Grouch was always one of my favorites. @cmrice-arch and @Tfranconi do you guys nickname things? As in, do you call Area 7 "Red Room" or such when you're speaking to each other about it? And/or in notes somewhere? (Might be a bad example, get away from the example.) The question arose really because of a project who did have numbers for their units, but never referred to them by number, always by names. If you do use nicknames, for what?

**Comment by lbestock** [comment]
@lrn5 Leah you are on here! I just didn't have your proper handle. This should send you an email, since I've tagged you, and then you can go ahead and add comments about nicknaming at Samothrace here - I am quite curious to hear more.

**Reply by lrn5** [reply]
I don't remember the exact conventions, but we start all the trench with "SGG.areaname.trenchnumber" and then every stratigraphic unit gets an SUyear.#. We definitely have nicknames for our trenches -- usually quite literally people names that we feel fit the character of the trench (admittedly strange), but they are never formally entered anywhere. We always use the official naming convention on any records. E.g. I believe we called one charlotte for a whole season for no particular reason other than it seeming to fit (not named after anyone of the team, truly a random name). It is either that or "trench #" in [area name].

The areas are somewhat defined by the traditional division of space in the sanctuary and in relation to where else we are digging.  Especially since we have only begun active excavation again 4 years ago, depending on how long you've been on the team/when you joined, people use very different terminology, but it is mostly based around architecture as reference points, e.g. "south of stoa" or "theatre hill"

**Reply by lrn5** [reply]
Another thought on this this morning, @lbestock  -- I think perhaps some of the strict adherence to long, but explicit naming conventions is a reaction to most of us having spent so much time reading old excavation notebook (getting back to the 1940's) with unclear trench names/locations, and having to spend a lot of time trying to plot them out. I do wonder with nicknames how helpful they will be when the entire team has turned over and the memory of why they were called what, etc. has been lost.

**Comment by lbestock** [comment]
Yeah totally and that is one reason why I don't always manage to hide my frustration with nicknames :) But it's also where a system can help. If there is ever anywhere where things are referred to without their proper name/identifier, where they are just nicknames, then in the formal record it makes sense to include nicknames attached to the proper identifier. So that someone in future can sort it out when the memory is gone.

**Comment by cmrice-arch** [comment]
Hmm, we have many nicknames for restaurants, stores, and people (all good and well until we need to make a reservation and can’t remember the actual names of places…), but I can’t think of any for numbered contexts. We do have some rooms that we refer to by functional name rather than by number (e.g. biclinium instead of Room 40), but these should all be cross referenced in multiple plans and context sheets. 




Candace M. Rice
Assistant Professor of Archaeology and Classics

Joukowsky Institute for Archaeology and the Ancient World
Brown University | Box 1837 | 60 George Street | Providence, RI  02912 | USA

**Reply by lbestock** [reply]
Ah ha but that's it precisely! If you reference it in a context sheet, where? Do you write next to "Room 40" "biclinium"? Or in the description section of your context sheet do you say "this appears to be the biclinium because..."?  We want to be able to accommodate that in the digital equivalent of a context sheet - the difference between a name for something and a description or interpretation of something is the issue here.
